server/api/auth.py

```diff
@@ -61,17 +61,19 @@ def decorator(func):
 
     password = request.json.get('password')
     hashed_pwd = hashlib.sha256(password.encode('utf-8')).hexdigest()
-    auth_info = await auth_query.fetch_info_by_user_id(
+    auth_info = await auth_query.fetch_info_by_user_name(
         request.app.config.DB_CONN, request.json.get('id')
     )
     if auth_info is None or auth_info.get('hashed_password') != hashed_pwd:
         raise ApiUnauthorized("Unauthorized: Incorrect user id or password")
     token = utils.generate_apikey(
         request.app.config.SECRET_KEY,
-        request.json.get('id')
+        auth_info.get('user_id')
+        # request.json.get('id')
```

Comment:
Please remove the commented code. We typically only commit active code and rely on version control for preserving old code.

Reply:
I have missed the comment. Completely agree that no code comment should be checked in. Thanks for pointing this out.